machine/reference/inspect.md

```diff
@@ -61,8 +61,10 @@ For the most part, you can pick out any field from the JSON in a fairly
 straightforward manner.
 
 ```none
+{% raw %}
 $ docker-machine inspect --format='{{.Driver.IPAddress}}' dev
```

Comment:
A couple of days ago I tried fixing this, that's why I noticed the JS errors. Here's what I did:

1. jekyll clean
2. jekyll serve
3. Take not of the pages that had liquid warnings
4. Try to fix those warnings
3. jekyll serve
4. Test my changes.

I noticed that in some cases, the result was not rendering properly, there were some snippets where the `{{<field>}}` part was being removed from the html.
Since my change was essentially what you did, I wanted to inspect what html was being generated for those pages. That's when I noticed the JS error.

From what I could, when we had `{{.<field>}}` (it started with a dot), I would get JS erros. I tried fiddling with it but could not get it to work properly. In some situations the text out be removed from the generated html, in other cases it would generate JS errors.

Maybe try the syntax suggested by @johndmulhausen, where you only escape what's inside the code fences.

    ```bash
    {% raw %}
    <sample>
    {% endraw %}
    ```